### ISSUE TITLE
[BUG]: (Vulkan) Host GPU lost too many times, device is probably completely wedged.

### ISSUE THREAD

**Issue by Zaur-Ismayilzada** [PCSX2/pcsx2#10886]
### Describe the Bug

As it is described above, this is specific to Vulkan. The game menu runs smoothly. Whenever the game loads the "first level", the whole program crashes abruptly for the most of the time. Rarely an error window pops up. Only in this case I am able to save a log file.

![error](https://github.com/PCSX2/pcsx2/assets/78559092/cfa5a123-aa9d-4073-b5ce-3ab5fd6b2ac9)

### Reproduction Steps

Play any game

### PCSX2 Revision

v1.7.5590

### Operating System

Windows 10 (64bit)

### CPU

11th Gen Intel(R) Core(TM) i3-1115G4 @ 3.00GHz (4 CPUs)

### GPU

Intel(R) UHD Graphics

### GS Settings

![gs_1](https://github.com/PCSX2/pcsx2/assets/78559092/d5f1ddb8-cf09-409a-9916-2034920a7031)
![gs_2](https://github.com/PCSX2/pcsx2/assets/78559092/deadcc99-2807-405a-b6cb-8a690fda9bb5)

### Logs & Dumps

[log1.txt](https://github.com/PCSX2/pcsx2/files/14490759/log1.txt)

**Comment by RedDevilus** [comment]
Intel UHD series from what I recall have awful Vulkan Drivers, there is a reason why we use Automatic Renderer, if it was Xe series instead it should be fine. Don't think we can do anything about it as it's on Intel.

**Comment by JordanTheToaster** [comment]
Install the latest Intel drivers for your iGPU and you can use Vulkan without issue. You can find them here.

https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html

**Comment by Zaur-Ismayilzada** [comment]
@JordanTheToaster 
Thanks for the reply. I regularly update my drivers. I am well aware of this website. After registering my product on Intel's website I got this
![image](https://github.com/PCSX2/pcsx2/assets/78559092/7b9d7616-3840-447c-833b-dbca01e831da)
I also registered my product on Lenovo's website. But there is obviously no update for my drivers.

I don't want my graphics card's default configurations to be messed up. That's why, I will not update it through Intel's website

Thanks again
